### PR TITLE
ci: add apic ext cli pipline

### PR DESCRIPTION
**Pull request by frankqianms**

### CODE REVIEW

File: .github/workflows/cli.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: CLI
```
Comment (blackchoey) [inline .github/workflows/cli.yml:1]: Give it a better name like CLI weekly live test

File: .github/workflows/cli.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+
+    steps:
+      - uses: actions/checkout@v2
```
Comment (blackchoey) [inline .github/workflows/cli.yml:16]: minor: we may not need to checkout this repo's code?

File: .github/workflows/cli.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - uses: azure/login@v2
```
Comment (blackchoey) [inline .github/workflows/cli.yml:23]: May need to add below content to make the login work (from https://github.com/marketplace/actions/azure-login#login-with-openid-connect-oidc-recommended)
```
permissions:
  id-token: write
  contents: read
```

File: .github/workflows/cli.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+on:
+  schedule:
+    - cron: "0 18 * * 0"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```
Comment (wenytang-ms) [inline .github/workflows/cli.yml:12]: if the python version is single, no need to setup as a matrix

**Pull request closed without merging**